app/models/item.rb

```diff
@@ -25,4 +25,11 @@ def no_orders?
     item_orders.empty?
   end
 
+  def self.most_popular
+    select('items.*, SUM(quantity) as total_quantity').joins(:item_orders).group('items.id').order('total_quantity DESC').limit(5)
+  end
+
+  def self.least_popular
+    select('items.*, SUM(quantity) as total_quantity').joins(:item_orders).group('items.id').order('total_quantity').limit(5)
+  end
```

Comment:
It probably is a little redundant to have identical methods only for `DESC` to be changed. Helper method?